docs/lib/datastore/fragments/android/getting-started.md

```diff
@@ -1,10 +1,10 @@
 ## Model Generation
 
-The first step to integrate DataStore into your app is to model your data with a GraphQL schema, and *generate models* from that schema. The same GraphQL may be used to generate models for Android, iOS, and JavaScript. The same schemas are also used by the network protocol when synchronizing data with the cloud. Models can be generated by running a Gradle task in Android Studio, or by executing a command with the Amplify CLI.
+The first step towards integrating DataStore into your app is to model your data. To do so, you'll define a GraphQL schema. Amplify will then *generate Java classes* from that schema. The same GraphQL may be used to generate models for Android, iOS, and JavaScript. The same schema are also used by the DataStore's network protocol when synchronizing data with the cloud. Models can be generated by running a Gradle task in Android Studio, or by executing a command with the Amplify CLI.
```

Comment:
"The same schema is also" perhaps?

Reply:
It's also weird how we start two consecutive statements with "the same." Should GraphQL be a noun? Or should this be consolidated into:

> The same schema may be used to generate models across Android, iOS, and JAvaScript, and is used by DataStore's network protocol when synchronizing data with the cloud.

Reply:
Random though: is it stylized DataStore? IIRC, the left navigation says Datastore right now.

Reply:
I like the compound you suggest and will use it.

`DataStore` with a capital `S` is the stylization that is used in the source code. I think we should update the left-nav, to the same.